plans/done/20260801-commit-graph-sidebar.md

```
@@ -0,0 +1,356 @@
+  { commits: RawGraphCommit[]; refs: GraphRefRecord[]; worktrees: WorktreeRecord[]; mergedBranches: string[]; totalCommits: number | null }
+>
+```
+
+Every git spawn happens in the worker (base resolution included — `resolveBaseComparison` is already in
+the worker's import graph via `getGitStatusSnapshot`), so both stdout draining and parsing stay off the
+host event loop. Spawns per call: `for-each-ref` (heads+remotes+tags, one call), `worktree list
+--porcelain`, `branch --merged`, `log`, `rev-list --count`, plus base resolution.
+
+Log format — subject last so embedded tabs can't shift fields:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)